masterlist.yaml

```diff
@@ -1200,62 +1428,97 @@ plugins:
       - Invent
       - Relev
   - name: 'Definitive Hearthfire Supllies.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/5993/']
     tag:
       - Invent
   - name: 'JS Armored Circlets SE.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/2140/']
     tag:
       - Relev
   - name: 'Deadly Wenches.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/599/']
     tag:
       - Delev
       - Relev
   - name: 'CraftingSupplies_AIO_ALL.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/2359/']
     tag:
       - Invent
   - name: 'Ducks and Swans.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/2149/']
     global_priority: -90
   - name: 'BirdsOfSkyrim_SSE.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/3097/']
     after:
       - 'SkyTEST-RealisticAnimals&Predators.esp'
       - 'skyBirds_SSE.esp'
   - name: 'more idle markers.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/3211/'
+        name: 'More Tavern Idles on Nexus Mods'
     after:
       - 'Immersive Citizens - AI Overhaul.esp'
   - name: 'Hunters Not Bandits.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/1547/']
     after:
       - 'Relationship Dialogue Overhaul.esp'
   - name: 'My Home Is Your Home.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/7096/']
     global_priority: 60
     after:
       - 'AmazingFollowerTweaks.esp'
   - name: 'CFTO.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/8379/'
+        name: 'Carriage and Ferry Travel Overhaul on Nexus Mods'
     after:
       - 'Immersive Citizens - AI Overhaul.esp'
   - name: 'FlowerGirls SE.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/5941/']
     after:
       - 'Relationship Dialogue Overhaul.esp'
   - name: 'Honeyside.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/918/'
+        name: 'Sweet as Honeyside on Nexus Mods'
     after:
       - 'ELFX - NoHoneyside.esp'
       - 'Alternate Start - Live Another Life.esp'
   - name: 'Immersive Sounds - Compendium.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/523/']
     after:
       - 'Reverb and Ambiance Overhaul - Skyrim.esp'
   - name: 'Landscape Fixes For Grass Mods.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/9005/']
     after:
       - 'Verdant - A Skyrim Grass Plugin SSE Version.esp'
   - name: 'DynDOLOD.esp'
+  # Dynamic Distant Objects LOD
```

Comment:
No need for the comment since you've added named links.

Reply:
Also removed.